Dockerfile

```diff
@@ -35,6 +35,9 @@ RUN apt-get update && apt-get install -y xauth libwebkitgtk-1.0-0
 RUN pan.sh -file ./plugins/platform-utils-plugin/samples/showPlatformVersion.ktr \
 	&& kitchen.sh -file samples/transformations/files/test-job.kjb
 
+# Copy plugins to the plugins directory
+COPY ./plugins/ plugins/
```

Comment:
Rather than supporting forks as variants, would it be a more standard workflow for variants to build a dependent image that uses this image in `FROM`? Additional plugins would be added as part of the dependent Dockerfile.

Reply:
That was my original plan. I started down that path with another image building off of this image to include plugins but then the makefile and spoon file which now use the .env image override are in the original repo, not in whatever repo you built your new image in. So at that point you need to go back to the original repo to copy the spoon file at least, maybe the makefile. Overall it just started to feel messy as I went down that road and I decided to go back to just having a branch with my plugins bundled in so that the .env file, makefile, and spoon file all worked as expected. Does that make sense, or do you think the plugins should still be in a dependent image, maybe just within this repo? Both images could be built and tagged appropriately and pushed to dockerhub.

Reply:
I ended up figuring out how to make this work with dependent images, which makes more sense. They are now just going to be in this repo and defined in the .env file. Still built using the make command. Seems like a better solution. Still not perfect, but better, probably.